go.mod

```diff
@@ -74,7 +74,7 @@ require (
 	github.com/shopspring/decimal v1.4.0
 	github.com/smartcontractkit/chain-selectors v1.0.23
 	github.com/smartcontractkit/chainlink-automation v1.0.4
-	github.com/smartcontractkit/chainlink-ccip v0.0.0-20240930150148-1c731b9602dd
+	github.com/smartcontractkit/chainlink-ccip v0.0.0-20240930194309-f49d68904bdf
```

Comment:
Change back to main once https://github.com/smartcontractkit/chainlink-ccip/pull/171 is merged.

Reply:
done.